endtoend/components/validator.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/testutil"
 )
 
+const depositGasLimit = 500000
```

Comment:
Is it ok that original amount was 4000000?

Reply:
The deposit contract has a ceiling of gas usage of around 500K, maybe I should leave it as is though :+1: